doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1112,6 +1112,50 @@ co-inductive definitions are also allowed.
    object of type :g:`(EqSt s1 s2)`. We will see how to introduce infinite
    objects in Section :ref:`cofixpoint`.
 
+Caveat
+++++++
+
+The ability to define co-inductive types by constructors, hereafter called
+*positive co-inductive types*, is known to break subject reduction. It is a
+fundamental issue related to the duality of computational behavior between
```

Comment:
I guess that it is the "fundamental issue" sentence that @herbelin would like you to state differently...